feature/office/src/main/java/com/iteneum/office/presentation/ui/OfficeView.kt

```diff
@@ -50,55 +77,51 @@ fun OfficeView() {
  * @param onEmailButtonClicked launches the Email intent
  * @author Yaritza Moreno
  * Modified by:
- * @author Andres Ivan Medina
+ * @modifiedBy Andres Ivan Medina
  */
-@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun OfficeUI(
-    officeData: OfficeModel,
+    officeData: Office,
     onCallButtonClicked: () -> Unit,
     onEmailButtonClicked: () -> Unit,
 ) {
-    Scaffold(content = { paddingValues ->
+    Column(
+        modifier = Modifier.fillMaxSize()
+    ) {
         val sizes = LeasePertTheme.sizes
-        Box(modifier = Modifier.padding(paddingValues)) {
-            Column(
-                Modifier.fillMaxSize()
-                //.padding(top = sizes.extraSize18)
-            ) {
-                LPTitleLarge(label = stringResource(
-                    id = R.string.LPInfo
-                ),
-                    color = MaterialTheme.colorScheme.tertiary,
-                    textAlign = TextAlign.Justify,
-                    weight = FontWeight.Bold,
-                    onClick = {})
-                LPBodyMedium(modifier = Modifier.padding(top = sizes.extraSize10),
-                    label = officeData.address,
-                    color = MaterialTheme.colorScheme.onPrimaryContainer,
-                    textAlign = TextAlign.Justify,
-                    weight = FontWeight.Normal,
-                    onClick = {})
-                LPBodyMedium(modifier = Modifier.padding(top = sizes.extraSize10),
-                    label = stringResource(id = R.string.LPHours) + officeData.schedule,
-                    color = MaterialTheme.colorScheme.onPrimaryContainer,
-                    textAlign = TextAlign.Justify,
-                    weight = FontWeight.Normal,
-                    onClick = {})
-                LpOutlinedButton(Modifier.padding(top = sizes.extraSize10),
-                    icon = Icons.Filled.Call,
-                    textButton = stringResource(id = R.string.LPCallButton),
-                    onClicked = {
-                        onCallButtonClicked()
-                    })
-                LpOutlinedButton(Modifier.padding(top = sizes.extraSize10),
-                    icon = Icons.Outlined.Mail,
-                    textButton = stringResource(id = R.string.LPMailButton),
-                    onClicked = {
-                        onEmailButtonClicked()
-                    })
+        LPTitleLarge(label = stringResource(
+            id = R.string.LPInfo
+        ),
+            color = MaterialTheme.colorScheme.tertiary,
+            textAlign = TextAlign.Justify,
+            weight = FontWeight.Bold,
+            onClick = {})
+        LPBodyMedium(modifier = Modifier.padding(top = sizes.extraSize10),
+            label = officeData.address,
```

Comment:
in named param **label** add stringResource() and try add **officeData.address** as stringResource() parameter.

Here is an example
![image](https://github.com/irv205/droid_leaspert/assets/33376944/6978e00b-ed4b-432d-bd58-08055ef47d9d)

Reply:
added:
val address = stringResource(id = R.string.LPAddress,officeData.address)
val schedule = stringResource(id = R.string.LPHours,officeData.schedule)